EasyCommands.Tests/ParameterParsingTests/AggregateBlockPropertyProcessorTests.cs

```diff
@@ -10,6 +11,11 @@ namespace EasyCommands.Tests.ParameterParsingTests {
     public class AggregateBlockPropertyProcessorTests {
         List<object> aggregationList = new List<object> { 1, 2, 3 };
 
+        [TestInitialize]
+        public void InitializeTestClass() {
+              System.Threading.Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US");      
+        }
+
```

Comment:
Making this change in all current (and future) tests is pretty heavy handed.  Is there a way to set this globally so that it's the case for all tests, in a way that doesn't have to be maintained for all future tests?

Reply:
It might be possible to hide the TestInitialize method in an abstract base. 
One would still need to remeber to derive future test from this base but the cognitive burden should be lower.